pkgdb/src/buildenv/realise.cc

```diff
@@ -286,6 +294,31 @@ getRealisedPackages( nix::EvalState &                   state,
   /* Interpret ooutput as derivation. */
   auto package_drv = getDerivation( state, *output.value, false );
 
+  auto broken = package_drv->queryMetaBool( "broken", false );
```

Comment:
note (blocking to at least create a ticket):
The man page says:
```
`allow.unfree`
:   Allows packages with unfree licenses to be installed and appear in search
    results.
    The default is `false`.
```

Shouldn't we be doing this filtering at resolve time? And probably search time as well (although that's less clear to me). If not we need to change docs

Reply:
Filtereing and respecting the `unfree` attribute should probably _also_ be done at resolution time. Checking unfree and broken here is mainly to retain current behaviour as is. Making change to resolution and broader behavior was not in scope for this PR IMO

Reply:
For unfree I created https://github.com/flox/flox/issues/1091

For broken it looks like we actually do filter those out for search/resolve, so doesn't that mean the conditional here should be unreachable?

Reply:
We'll need a followup for this either way so I don't care if we address later